backend/chalice/api_server/app.py

```diff
@@ -22,6 +22,16 @@
 from corpora.common.utils.aws_secret import AwsSecret
 from corpora.common.corpora_config import CorporaAuthConfig
 
+ALLOWED_ORIGINS = [
+    "http://localhost:8000",
+    "http://localhost:9000",
```

Comment:
In the future, could we check the env of the server and only append localhost origins if it is dev or staging? Just to prevent other clients from accessing prod.

Reply:
Yes! @Bento007 is helping to investigate a complete solution 👌 This change is a temp measure to ease development!